src/test/java/org/activiti/cloud/query/ApplicationIT.java

```diff
@@ -5,10 +5,12 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = {QueryApplication.class})
 @DirtiesContext
```

Comment:
you probably don't need this `@DirtiesContext` and in general we should get rid of these because when present, it tends to be slower